.github/workflows/lint.yml

```
@@ -96,3 +96,14 @@ jobs:
         echo '::group::Lint Python types'
         mypy --install-types --non-interactive --config-file mypy.ini
         echo '::endgroup::'
+
+  bc:
+    if: github.event.pull_request
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run BC Lint Action
+        uses: pytorch/test-infra/.github/actions/bc-lint@main
```

Should we maybe switch to fixed version here, i.e.

```suggestion
        uses: pytorch/test-infra/.github/actions/bc-lint@v20230509-105021
```

or

```suggestion
        uses: pytorch/test-infra/.github/actions/bc-lint@d8944a4b2a96b70f66c3c56d97b93c2c6bc9936a
```

We are pinning the linter versions for everything else as well to avoid issues coming from upstream.

---

IMO keep main so we can benefits from improvements automatically. If it becomes noisy because of failures we can revisit